sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/persist/fileAccessControl.ts

```diff
@@ -165,7 +165,7 @@ export class FileAccessControl {
       if (psProc.error) {
         throw psProc.error;
       } else if (psProc.status !== 0) {
-        throw new Error(`Getting ACL identity did not succeed (PS returned code ${psProc.status})`);
+        throw new Error(`Getting ACL identity did not succeed (PS returned code ${psProc.status!})`);
```

Comment:
Should the "(PS returned code ${psProc.status})" be printed only if `psProc.status` is defined?

Reply:
Yes, adding a `null` check now.